### DISCUSSION TITLE
Airflow Stable API v1 | PermissionDenied even with auth.backend.default

### DISCUSSION THREAD

**Discussion by Madaditya** [apache/airflow#13219] [Q&A] [answered]
I have been trying to build on top of [Airflow 2.0.0 docker image](https://hub.docker.com/layers/apache/airflow/2.0.0-python3.6/images/sha256-e973fef20d3be5b6ea328d2707ac87b90f680382790d1eb027bd7766699b2409?context=explore) and get the scheduler + webserver running to access and test the [stable API](https://airflow.apache.org/docs/apache-airflow/stable/stable-rest-api-ref.html)

However, whenever I try and test the API it gives me a forbidden error.

> root@afce172a35c2# _airflow config get-value api auth_backend_
> 
> **airflow.api.auth.backend.default**
> 
> root@afce172a35c2# _curl localhost:8080/api/v1/dags_
> {
>   "detail": null,
>   "status": 403,
>   "title": "**Forbidden**",
>   "type": "https://airflow.apache.org/docs/2.0.0/stable-rest-api-ref.html#section/Errors/PermissionDenied"
> }

**airflow.api.auth.backend.default** should disable any authentication on the API as [per documentation](https://airflow.apache.org/docs/apache-airflow/stable/security/api.html)

Any help is appreciated!

**Comment by kaxil** [comment, marked as answer]
That is only true for experimental API.

You can't disable auth for stable API.

>If you wish to have the experimental API work, and aware of the risks of enabling this without authentication

**Reply by kaxil** [reply]
https://airflow.apache.org/docs/apache-airflow/stable/security/api.html#disable-authentication
